great_expectations/core/usage_statistics/anonymizers/data_connector_anonymizer.py

```diff
@@ -0,0 +1,52 @@
+from great_expectations.core.usage_statistics.anonymizers.anonymizer import Anonymizer
+from great_expectations.datasource.data_connector import (
+    ConfiguredAssetFilePathDataConnector,
+    ConfiguredAssetFilesystemDataConnector,
+    ConfiguredAssetS3DataConnector,
+    ConfiguredAssetSqlDataConnector,
+    DataConnector,
+    FilePathDataConnector,
+    InferredAssetFilePathDataConnector,
+    InferredAssetFilesystemDataConnector,
+    InferredAssetS3DataConnector,
+    InferredAssetSqlDataConnector,
+    RuntimeDataConnector,
+)
+
+
+class DataConnectorAnonymizer(Anonymizer):
+    def __init__(self, salt=None):
+        super().__init__(salt=salt)
+
+        # ordered bottom up in terms of inheritance order
+        self._ge_classes = [
+            InferredAssetS3DataConnector,
+            InferredAssetFilesystemDataConnector,
+            InferredAssetFilePathDataConnector,
+            InferredAssetSqlDataConnector,
+            ConfiguredAssetS3DataConnector,
+            ConfiguredAssetFilesystemDataConnector,
+            ConfiguredAssetFilePathDataConnector,
+            ConfiguredAssetSqlDataConnector,
+            RuntimeDataConnector,
+            FilePathDataConnector,
+            DataConnector,
+        ]
```

Comment:
This list will need to be modified whenever we write a new `DataConnector` class. A non-blocking request : could we add a comment here so that it's easy to navigate to?